### PR TITLE
Feat/add new module wsus store updates locally

### PR DESCRIPTION
**Pull request by raumdonut**
Added the module wsus_store_updates_locally. This is used to define if the Windows Update Files are stored locally on the wsus or the clients recieve them via Microsoft Update.
Read more here: https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ms744600(v=vs.85)

### CODE REVIEW

File: plugins/modules/wsus_store_updates_locally.ps1 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        update_location = @{ type = "str"; choices = "local", "microsoft_update"; default = "local" }
+        state           = @{ type = "str"; choices = "absent", "present"; default = "present" }
+    }
+    supports_check_mode = $false
+}
+
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
+
+# Map variables
+$updatelocation = $module.Params.update_location
```
Comment (d-strobel) [inline plugins/modules/wsus_store_updates_locally.ps1]: name of variables should be camelcase

**Comment by d-strobel** [comment]
I think the name of the module is not appropriate.
How about "wsus_store_update_location" ?

**Comment by raumdonut** [comment]
> I think the name of the module is not appropriate. How about "wsus_store_update_location" ?

Yes that sounds a lot better. Will be fixed next commit.

File: plugins/modules/wsus_store_updates_locally.ps1 (hunk 1 above)
Comment (raumdonut) [reply plugins/modules/wsus_store_updates_locally.ps1]: Is fixed in latest commit

**Pull request merged**